src/libs/ReportUtils.ts

```diff
@@ -2707,6 +2715,80 @@ function canEditReportAction(reportAction: OnyxEntry<ReportAction>): boolean {
     );
 }
 
+function getParentReportAction(parentReportActions: OnyxEntry<ReportActions>, parentReportActionID: string | undefined): OnyxEntry<ReportAction> {
```

Comment:
I thought you mentioned you removed this one?

Reply:
@robertjchen This is my concern, we should use deprecated function or creating a new one here?

Reply:
Is there a reason why we can't use `Use Onyx.connect() or withOnyx() instead` as recommended by the deprecation comment? 🤔

Reply:
@Krishna2323 Bump on the above question.


I checked `ReportUtils`, I saw that we're still using `ReportActionsUtils.getParentReportAction(report);` to get `parentReportActions`.

Reply:
little bump @Krishna2323 so we can merge this one.

Reply:
What are your thoughts on this, @robertjchen? Since we're already using the deprecated `getParentReportAction` in `ReportUtils`, I'm okay with using it until we have a plan to remove it all at once.

Reply:
Sounds good, given the current progress on the migration- we can use it for now 👍

Reply:
Thanks for confirming 😄 .
@Krishna2323 Can you update this please? Thank you 🙇